portfolio/src/test/java/com/google/sps/servlets/CommentsServletTest.java

```diff
@@ -0,0 +1,43 @@
+package com.google.sps.servlets;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.gson.Gson;
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+class CommentsServletTest {
+  private CommentsServlet servlet;
+  private MockHttpServletRequest request;
+  private MockHttpServletResponse response;
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+  @BeforeEach
+  public void setUp() {
+    servlet = new CommentsServlet();
+    request = new MockHttpServletRequest();
+    response = new MockHttpServletResponse();
+    helper.setUp();
+  }
+
+  @Test
+  void testPost() {
+    Gson gson = new Gson();
+    Comment comment = new Comment();
```

Comment:
This is not used here, forgot to remove it. It is removed in the further PRs though, so I hope it is okay